portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,42 +60,55 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     List<Entity> commentsList = results.asList(FetchOptions.Builder.withLimit(parameterValue));
 
+    
+
     for (Entity entity : commentsList) {
         long id = entity.getKey().getId();
         String comment = "";
         String email = "";
+        String url = "";
+        long timestamp = 0;
+
         if(entity.getProperty("comment") instanceof String){
             comment = (String) entity.getProperty("comment");
         } else {
             System.err.println("Can't load comments");
+            System.out.println("comment error");
         }
 
-        long timestamp = 0;
         if(entity.getProperty("timestamp") instanceof Long){
             timestamp = (Long) entity.getProperty("timestamp");
         } else {
             System.err.println("Can't load comments");
+            System.out.println("timestamp error");
         }
 
         if(entity.getProperty("email") instanceof String){
             email = (String) entity.getProperty("email");
         } else {
             System.err.println("Can't load comments");
+            System.out.println("email error");
         }
 
-        
-        Comment commentTask = new Comment(id, comment, timestamp, email);
+        if(entity.getProperty("url") instanceof String){
+            url = (String) entity.getProperty("url");
+        } else {
+            System.err.println("Can't load comments");
+            System.out.println("url error");
+        }
+
+        Comment commentTask = new Comment(id, comment, timestamp, email, url);
+
         comments.add(commentTask);
```

Comment:
minor thing, but it would be OK to just do

`comments.add(new Comment(id, comment, timestamp, email, url));`

since you're not reusing the variable and it's clear what's happening.